### DISCUSSION TITLE
It is hoped that this software will add the function of flashing drawn graphics.

### DISCUSSION THREAD

**Discussion by bdbox1** [pubpub-zz/ppInk#299] [Ideas]
In order to attract attention, it is hoped that this software will add the function of flashing drawn graphics. For example, 1) after drawing a line, 2) select the graph, 3) then click a button, and 4) the selected graph flashes.
It is best to click on the flashing function to be able to use the shortcut key.

**Comment by pubpub-zz** [comment]
you can do flashing drawing using APNG. I think that's sufficient. adding such a feature adds some complexity. I will not spend time on this. Feel free to propose a PR. Without update I will convert this issue as discussion/idea
